openmrs-19/src/main/resources/task-data-provider.json

```diff
@@ -14,7 +14,7 @@
       ],
       "fields": [
         {
-          "displayName": "openMRS.provider.providerId",
+          "displayName": "openMRS.provider.id",
           "fieldKey": "providerId"
```

Comment:
Sometimes the fieldKey value is in the form "person.id" and sometimes it's in the form "providerId", do you think we should make it consistent (either camelCase or period.delimited)?